homeassistant/const.py

```diff
@@ -770,6 +773,8 @@ class UnitOfEnergy(StrEnum):
     WATT_HOUR = "Wh"
     KILO_WATT_HOUR = "kWh"
     MEGA_WATT_HOUR = "MWh"
+    GIGA_WATT_HOUR = "GWh"
+    TERRA_WATT_HOUR = "TWh"
```

Comment:
```suggestion
    TERA_WATT_HOUR = "TWh"
```

Reply:
Yes, of course - db99bd69ec21fe28ca2bd86b9a1bb4544686efe4